config/m4/mpi.m4

```diff
@@ -38,4 +40,5 @@ AS_IF([test -n "$MPICC"],
       [mpi_enable=disabled])
 AM_CONDITIONAL([HAVE_MPI],    [test -n "$MPIRUN"])
 AM_CONDITIONAL([HAVE_MPICC],  [test -n "$MPICC"])
+AM_CONDITIONAL([HAVE_SHMEMCC],[test -n "$SHMEMCC"])
```

Comment:
need to define it also in https://github.com/openucx/ucx/blob/83a23dcecd117f7daef3e37136f7e730d4adba9f/configure.ac#L223
when UCX is built with `--with-docs-only`

Reply:
Ah..I will fix